### ISSUE TITLE
cf copy-source -h incorrect

### ISSUE THREAD

**Issue by Zteve** [cloudfoundry/cli#779]
### Command
`cf copy-source -h`
### CLI Version
6.16.0+ee4d916
### Error
`cf copy-source SOURCE-APP TARGET-APP [-o TARGET-ORG] [-s TARGET-SPACE] [--no-restart]`
*should* read:
`cf copy-source SOURCE-APP TARGET-APP [ [-o TARGET-ORG] -s TARGET-SPACE] [--no-restart]`
since the space must be supplied if a target organization is supplied.
### Stack Trace
N/A
### CC API Version
N/A
### CF Trace
N/A
### Platform Details
Mac OS X 10.11.3
### Shell
Terminal

**Comment by krishicks** [comment]
The line above doesn't seem quite right.

If anything, we should make it:

```
cf copy-source SOURCE-APP TARGET-APP [-o TARGET-ORG -s TARGET-SPACE] [--no-restart]
```

And we should add examples for copying an app into the same org/space and into a different org/space.

**Comment by Zteve** [comment]
But your example doesn't allow a different space in the current organization to be specified, which is currently allowed.

**Comment by dkoper** [comment]
What you're describing is what our implementation is and will remain to do.
What I think @krishicks is describing is a way to explain it without causing people to have to read it multiple times and count the brackets.

**Comment by Zteve** [comment]
Thank you.  Surely  it would be better to tell the truth?

**Comment by krishicks** [comment]
I hadn't considered the case where you copy to a different space in the same org.

Given that possibility, I'll vote in favor of @Zteve's suggestion and drop my own.

I still think there needs to be a lengthier description so we don't have to rely on people parsing the groups of brackets to understand how they can call the command.

**Comment by Zteve** [comment]
Not going to complain about that.

Steve Powell          [M: +*44*-*7815*-*838*-*558* H:+*44*-*1962*-*775*-
*598*]
  *Every great art must be supplemented by leisurely discussion, by
stargazing, if you will, about the nature of things.*
Plato, *Phædrus*
pivotal.io <http://www.pivotal.io>|SpringSource
<http://www.springsource.org>|RabbitMQ <http://www.rabbitmq.com>

**Comment by dkoper** [comment]
The team suggested the following - basically just swapping the two options in your suggestion to improve readability:

`cf copy-source SOURCE-APP TARGET-APP [-s TARGET-SPACE [-o TARGET-ORG]] [--no-restart]`

**Comment by Zteve** [comment]
This is a good suggestion. +1

**Comment by dkoper** [comment]
This has been fixed in cf CLI v6.17.0.
https://github.com/cloudfoundry/cli#downloads

Thanks again!
Dies Koper
CF CLI PM
